### PR TITLE
Project renaming XP-225

### PR DESCRIPTION
**Pull request by tanertopal**

### CODE REVIEW

**Comment by atymoshchuk** [comment]
Do we want to name our custom logger as `xain logger`?

File: RELEASE_PROCESS.md [hunk 1]
```diff
@@ -17,26 +17,26 @@ A release on git is just a tagged commit on the `master` branch.
...
-    - updates the
-      [`CHANGELOG.md`](https://github.com/xainag/xain/blob/master/CHANGELOG.md)
-      with all notable changes for the release.
-    - possibly update the `Development Status` classifiers in the
-      [`setup.py`](https://github.com/xainag/xain/blob/master/setup.py). You
-      can check supported classifiers in the [pypi
-      website](https://pypi.org/classifiers/).
-2. Got to the [Github Releases tab](https://github.com/xainag/xain/releases)
+   - increases the version number in
+     [`xain/__version__.py`](https://github.com/xainag/xain-fl/blob/master/xain_fl/__version__.py)
```
Comment (Robert-Steiner) [inline RELEASE_PROCESS.md]: `xain_fl/version`

**Comment by tanertopal** [comment]
@Robert-Steiner and @atymoshchuk requested changes applied

**Comment by atymoshchuk** [comment]
1. Please change path in comment here: https://github.com/xainag/xain/blob/tanertopal/project_renameing-XP-225/xain_fl/datasets/stats/__main__.py#L19
2.  Would be better to be more generic here: https://github.com/xainag/xain/blob/tanertopal/project_renameing-XP-225/xain_fl/logger.py#L1
For example: `This module contains custom logging configuration` and Line 8 accordingly.

**Comment by atymoshchuk** [comment]
Can you please double check Readme file and correct project name as well, thanks.

File: README.md [hunk 2]
```diff
@@ -46,7 +47,7 @@ For development we require some extra system dependencies:
 To clone this repository and to install the XAIN project, please execute the following commands:
 
 ```shell
-$ git clone https://github.com/xainag/xain.git
+$ git clone https://github.com/xainag/xain-fl.git
 $ cd xain
```
[the remark this answers is not in the document]
Comment (tanertopal) [reply README.md]: I'd rather rename it now and than rename the repo directly after merging the PR

**Pull request merged**